### PR TITLE
chore: lightpush tests coverage improvement

### PR DESCRIPTION
**Pull request by romanzac**
Description
Batch of tests for Waku LightPush v2 to improve coverage

Changes
Tests available to run:
cd go-waku/waku/v2/protocol/lightpush
go test

### CODE REVIEW

File: waku/v2/protocol/lightpush/waku_lightpush_test.go [hunk 1]
```diff
@@ -226,3 +227,100 @@ func TestWakuLightPushAutoSharding(t *testing.T) {
...
+		defer wg.Done()
+		<-sub2.Ch
+	}()
+
+	var lpOptions []Option
+	lpOptions = append(lpOptions, WithPubSubTopic(testTopic))
+	lpOptions = append(lpOptions, WithPeer(host2.ID()))
+
+	// Check that msg publish has passed for nominal case
+	_, err = client.Publish(ctx, msg2, lpOptions...)
```
Comment (chaitanyaprem) [inline waku/v2/protocol/lightpush/waku_lightpush_test.go:297]: We should check if published message is received by node1 and if not fail the test.

**Review by chaitanyaprem — APPROVED** [review]
Looks good apart from minor comment.

File: waku/v2/protocol/lightpush/waku_lightpush_test.go (hunk 1 above)
Comment (romanzac) [reply waku/v2/protocol/lightpush/waku_lightpush_test.go:303]: Addressed at https://github.com/waku-org/go-waku/pull/957/commits/7973079e09f425784f47599cd7b71ae08c29d618 Please have a look.

**Review by chaitanyaprem — APPROVED** [review]
LGTM

**Review by richard-ramos — APPROVED** [review]

**Pull request merged**